cmd/fabric8-jenkins-proxy/main.go

```diff
@@ -111,6 +112,24 @@ func start(config configuration.Configuration, tenant *clients.Tenant, wit clien
 	wg.Wait()
 }
 
+func listenAndServeHelper(srv *http.Server, cancel context.CancelFunc) {
+	IsSslEnabledStr, _ := os.LookupEnv("JC_ENABLE_SSL")
+	bSslEnabled, parseErr := strconv.ParseBool(IsSslEnabledStr)
+	if parseErr == nil {
+		if bSslEnabled {
```

Comment:
naming variables with this nomenclature reminds me some dark ages hahahaha but well probably it is fine.